chapters/functions.tex

```diff
@@ -2418,13 +2418,14 @@ \subsection{Annotations for External Libraries and Include Files}\label{annotati
                  Include = "#include \"ExternalFunc1.h\"",
                  SourceDirectory =
                   "modelica://ExternalFunctions/Resources/Source");
-      // The SourceDirectory is the default and thus redundant.
+      // This SourceDirectory is the default and thus redundant.
   end ExternalFunc1;
 
   function ExternalFunc2
     input Real x;
     output Real y;
   external "C"
+    y = ExternalFunc2_ext(x)
```

Comment:
The `external-function-call` is optional in the grammar, so leaving it out could have been intentional.

Reply:
The C function is also called `ExternalFunc2` (like it would if the external call was left out) in the header file shown further down in the section, so `ExternalFunc2_ext` is wrong in that regard.

Reply:
Right, I missed that part. `ExternalFunc2` is a `void` function in that header part. That should be a `double` for the return value, right?

Reply:
I agree that it is likely intentional that it is missing here. The first function has the declaration to show that it is possible. However, the C-header should have double return type.